yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -877,12 +877,12 @@ func fetchColumnsWithUnsupportedDataTypes() ([]utils.TableColumnsDataTypes, erro
 	return unsupportedDataTypes, nil
 }
 
-const ORACLE_PARTITION_DEFAULT_COLOCATION = `The Assessment Report provides sharding/colocation recommendations for each partition individually.<br>
-All partitions of a partitioned table are currently created as colocated by default, due to certain limitations. 
-To manually modify the schema for sharding, please refer: <a class="highlight-link" href="https://github.com/yugabyte/yb-voyager/issues/1581">https://github.com/yugabyte/yb-voyager/issues/1581</a>.`
+const ORACLE_PARTITION_DEFAULT_COLOCATION = `For sharding/colocation recommendations, each parition is treated indiviudally. During the export schema phase, all the partitions of a partitioned table are currently created as colocated by default. 
+To manually modify the schema, please refer: <a class="highlight-link" href="https://github.com/yugabyte/yb-voyager/issues/1581">https://github.com/yugabyte/yb-voyager/issues/1581</a>.`
```

Comment:
make sure to add `<br>` for a newline wherever required, otherwise i dont think html will pickup that